src/main/resources/templates/db_scripts/schema.json

```diff
@@ -34,12 +34,16 @@
     },
     {
       "tableName":"accounts",
-      "fromModuleVersion":"15.0",
+      "fromModuleVersion":"15.9.2",
       "withMetadata":true,
       "index" : [
         {
           "fieldName" : "loanId",
           "tOps" : "ADD"
+        },
+        {
+          "fieldName": "userId",
```

Comment:
Will this actually work for the query
lower(f_unaccent(accounts.jsonb->>'userId')) LIKE lower(f_unaccent('e96618a9-04ee-4fea-aa60-306a8f4dd89b'))
Does the warning actually disappear with this index?

Reply:
I would think so. I haven't tested it, but we did a similar thing for other indexes with the same type of warning.  I can try to test it if you want.